cmd/serverless-init/tag/tag.go

```diff
@@ -59,6 +59,8 @@ func GetBaseTagsMapWithMetadata(metadata map[string]string) map[string]string {
 		tags[key] = value
 	}
 
+	tags["origin"] = "cloudrun"
```

Comment:
Could you add a test for that? (I believe `tag_test.go` might fail in this PR)

Reply:
Good catch, totally forgot, done!